### PR TITLE
Deprecate check_in_experience_lorota_security_updates_enabled feature flag

### PR DESCRIPTION
**Pull request by brianseek**
## Summary
Removes `check_in_experience_lorota_security_updates_enabled` feature flag. Also removes all instances of last4 and makes DOB the standard validation input.

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#50857

## Testing done
Updates existing tests

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by wullaski — CHANGES_REQUESTED** [review]
🧹 🕸️  One thing I can think of that might have been missed, and this is only because I came across this yesterday, is that there is now and unused key in the translation files `were-sorry-we-couldnt-match-your-information-to-our-records-please-try-again`

I found a few more.
`please-enter-the-last-4-digits-of-your-social-security-number`
`last-4-digits-of-your-social-security-number`
`invalid-last4-or-last-name`

**Comment by wullaski** [comment]
I also found `withLorotaSecurityUpdate` in /Users/wullaski/Dev/vets-website/src/applications/check-in/api/local-mock-api/e2e/ApiInitializer.js that probably could be removed as well. The few tests that use it could switch to using `withCurrentFeatures` maybe.

**Comment by brianseek** [comment]
> 🧹 🕸️ One thing I can think of that might have been missed, and this is only because I came across this yesterday, is that there is now and unused key in the translation files `were-sorry-we-couldnt-match-your-information-to-our-records-please-try-again`
> 
> I found a few more. `please-enter-the-last-4-digits-of-your-social-security-number` `last-4-digits-of-your-social-security-number` `invalid-last4-or-last-name`

For translation keys, I think our plan is to allow them to be orphaned and clean them up at translation audits. I don't see these coming back but I intentionally left them there since we discussed that at some point.

**Comment by brianseek** [comment]
> I also found `withLorotaSecurityUpdate` in /Users/wullaski/Dev/vets-website/src/applications/check-in/api/local-mock-api/e2e/ApiInitializer.js that probably could be removed as well. The few tests that use it could switch to using `withCurrentFeatures` maybe.

Very good catch! I'll get this fixed up.

**Review by wullaski — APPROVED** [review]
🚢  🤡

File: src/applications/check-in/pre-check-in/tests/e2e/errors/post-session/error.in.body.cypress.spec.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ describe('Pre-Check In Experience ', () => {
 
       ValidateVeteran.attemptToGoToNextPage();
 
-      Error.validatePageLoaded();
+      Error.validatePageLoaded(true, true);
```
Comment (acrollet) [inline src/applications/check-in/pre-check-in/tests/e2e/errors/post-session/error.in.body.cypress.spec.js:34]: This seems a little bit weird - do the cypress fixtures default to sending phone appointments now?

**Review by acrollet — CHANGES_REQUESTED** [review]
This looks great and it's awesome to get so much stuff out! Just one change I don't understand in the e2e tests :)

File: src/applications/check-in/pre-check-in/tests/e2e/errors/post-session/error.in.body.cypress.spec.js (hunk 1 above)
Comment (brianseek) [reply src/applications/check-in/pre-check-in/tests/e2e/errors/post-session/error.in.body.cypress.spec.js:34]: This is a stop gap, John has an in-progress PR to address this.

**Review by acrollet — APPROVED** [review]

**Pull request merged**